docs/kubectl.md

```diff
@@ -450,9 +450,9 @@ Usage:
 
```

Comment:
> Line 449 is the one that needs to be changed.

Sorry, two branches and I missed that part of the change.

How about the new flag types for .Provided() and merging the values?